File: app/src/main/java/org/lulzm/waft/SosAdapter/Datum.java
package org.lulzm.waft.SosAdapter;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/*********************************************************
 *   $$\                  $$\             $$\      $$\   
 *   $$ |                 $$ |            $$$\    $$$ |  
 *   $$ |      $$\   $$\  $$ | $$$$$$$$\  $$$$\  $$$$ |  
 *   $$ |      $$ |  $$ | $$ | \____$$  | $$ \$\$$ $$ | 
 *   $$ |      $$ |  $$ | $$ |   $$$$ _/  $$  \$$  $$ |  
 *   $$ |      $$ |  $$ | $$ |  $$  _/    $$ | $  /$$ |  
 *   $$$$$$$$  \$$$$$$$ | $$ | $$$$$$$$\  $$ | \_/ $$ |  
 *   \_______| \______/   \__| \________| \__|     \__|  
 *
 * Project : WAFT                             
 * Created by Android Studio                           
 * Developer : Lulz_M                                    
 * Date : 2019-06-13 013                                        
 * Time : 오후 12:40                                       
 * GitHub : https://github.com/scadasystems              
 * E-mail : redsmurf@lulzm.org                           
 *********************************************************/
public class Datum {
    @SerializedName("Country")
    @Expose
    private Country country;
    @SerializedName("Ambulance")
    @Expose
    private Ambulance ambulance;
    @SerializedName("Fire")
    @Expose
    private Fire fire;
    @SerializedName("Police")
    @Expose
    private Police police;

    public Country getCountry() {
        return country;
    }

    public void setCountry(Country country) {
        this.country = country;
    }

    public Ambulance getAmbulance() {
        return ambulance;
    }

    public void setAmbulance(Ambulance ambulance) {
        this.ambulance = ambulance;
    }

    public Fire getFire() {
        return fire;
    }

    public void setFire(Fire fire) {
        this.fire = fire;
    }

    public Police getPolice() {
        return police;
    }

    public void setPolice(Police police) {
        this.police = police;
    }
}

File: app/src/main/java/org/lulzm/waft/SosAdapter/Datum.kt
package org.lulzm.waft.SosAdapter

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/*********************************************************
 * $$\                  $$\             $$\      $$\
 * $$ |                 $$ |            $$$\    $$$ |
 * $$ |      $$\   $$\  $$ | $$$$$$$$\  $$$$\  $$$$ |
 * $$ |      $$ |  $$ | $$ | \____$$  | $$ \$\$$ $$ |
 * $$ |      $$ |  $$ | $$ |   $$$$ _/  $$  \$$  $$ |
 * $$ |      $$ |  $$ | $$ |  $$  _/    $$ | $  /$$ |
 * $$$$$$$$  \$$$$$$$ | $$ | $$$$$$$$\  $$ | \_/ $$ |
 * \_______| \______/   \__| \________| \__|     \__|
 *
 * Project : WAFT
 * Created by Android Studio
 * Developer : Lulz_M
 * Date : 2019-06-13 013
 * Time : 오후 12:40
 * GitHub : https://github.com/scadasystems
 * E-mail : redsmurf@lulzm.org
 */
class Datum {
    @SerializedName("Country")
    @Expose
    var country: Country? = null
    @SerializedName("Ambulance")
    @Expose
    var ambulance: Ambulance? = null
    @SerializedName("Fire")
    @Expose
    var fire: Fire? = null
    @SerializedName("Police")
    @Expose
    var police: Police? = null
}
